src/states_screens/gp_info_screen.cpp

```diff
@@ -235,7 +235,7 @@ void GPInfoScreen::init()
 
         // Avoid negative numbers (which can happen if e.g. the number of karts
         // in a previous race was lower than the number of players now.
-        int num_ai = UserConfigParams::m_num_karts - race_manager->getNumLocalPlayers();
+        int num_ai = UserConfigParams::m_default_num_karts - race_manager->getNumLocalPlayers();
```

Comment:
Need to change this to use the dictionary, but ensure that there are no identical enum values (because one is for minor mod whilst the other is for major mode).